src/scripts.js

```diff
@@ -275,16 +386,17 @@ let updateWelcomeMessage = (user) => {
 let loadUserData = (user, userRepo) => {
   loadCardInfo(user, userRepo);
   updateWelcomeMessage(user);
-  displayDailyWaterConsumption(user1, today)
-  displayWeeklyWaterConsumption(user1, today)
-  displayAllTimeSleepData(user1)
-  displayDailySleepData(user1, otherToday)
-  displayWeeklySleepHours(user1, otherToday)
-  displayWeeklySleepQuality(user1, otherToday)
-  displayDailySteps(user1, otherToday)
-  displayDailyMiles(user1, otherToday)
-  displayWeeklyActivity(user1, otherToday)
-  displayDailyActivityVsAll(user1, otherToday)
+  displayDailyWaterConsumption(user, today)
+  displayWeeklyWaterConsumption(user, today)
+  displayDailyAndAverageSleepData(user, otherToday)
+  // displayDailySleepData(user, otherToday)
```

Comment:
Excited to comb over these and really make sure we are calling everything we need to, and may break some of these function invocations into more module sections to be loaded.